docs/user/dashboard/lens.asciidoc

```diff
@@ -139,6 +139,42 @@ image::images/lens_drag_drop_3.gif[Using drag and drop to reorder]
 
 . Press Space bar to confirm, or to cancel, press Esc.
 
+[float]
+[[lens-formulas]]
+==== Formulas
+
+Formulas let you perform math on aggregated data in Lens by typing
+math and quick functions. To access formulas,
+click the *Formula* tab in the dimension editor. Access the complete
+reference for formulas from the help menu.
```

Comment:
Can we add an image of the help button once it's finalized?

Reply:
Yes, it's still held up for now.